EWL-4657: footer menu link class

designerneil opened the pull request:
## Ticket(s)

**Jira Ticket**
- [EWL: 4657: SG2 | Footer menu links need to use atoms](https://issues.ama-assn.org/browse/EWL-4665)

## Description
Footer links need to use the link atom to ensure consistency throughout site.

## To Test
- [ ] `gulp serve`
- [ ] Go to http://localhost:3000/?p=organisms-footer
- [ ] Observe the footer links are not broken
- [ ] Go http://localhost:3000/?p=pages-topic
- [ ] Observe the footer links are not broken

## Visual Regressions

Nothing to note. The only thing I did notice was that the margins and padding keep coming up differently from what's been last deployed. Which is normal, because:
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/127
has not been deployed yet

## Relevant Screenshots/GIFs
<img width="1232" alt="screen shot 2018-03-02 at 1 51 11 pm" src="https://user-images.githubusercontent.com/2271747/36918836-cb62a1c6-1e20-11e8-8297-0c5d8fb8dd6c.png">


## Remaining Tasks
N/A


## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

designerneil committed:
EWL-4657: Update links in the footer to use link atoms

designerneil committed:
updates backstop test ref images

labbydev commented on styleguide/source/_patterns/03-organisms/footer.twig:
```diff
@@ -14,7 +14,7 @@
               {% for link in column %}
                 <li>
-                  <a href="{{ link.url }}" class="ama__link--black">{{ link.text }}</a>
+                  {% include "@atoms/link.twig" %}
```
Go ahead and set the class here so that its `{% include "@atoms/link.twig" with "class" : "ama__link--black" %}`

labbydev reviewed (changes requested):
@designerneil when hovering this is giving the `$purpleHover` background which I believe should be resolved. 

<img width="1013" alt="screen shot 2018-03-06 at 4 33 37 pm" src="https://user-images.githubusercontent.com/1653723/37062610-9501a200-215c-11e8-8554-bc00ad61ae44.png">

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4657-footer-menu-link-class

# Conflicts:
#	styleguide/backstop_data/bitmaps_reference/backstop_default_news_0_document_3_bp-large.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead-with-subtitle_0_ama__masthead_0_bp-xs.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead-with-subtitle_0_ama__masthead_1_bp-small.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead-with-subtitle_0_ama__masthead_2_bp-med.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead_0_ama__masthead_0_bp-xs.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead_0_ama__masthead_1_bp-small.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_organisms-masthead_0_ama__masthead_2_bp-med.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_0_bp-xs.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_1_bp-small.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_2_bp-med.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_3_bp-large.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_4_bp-xl.png

designerneil committed:
EWL: 4657: Adds link classes directly to footer links

designerneil committed:
EWL: 4657: Removes link classes directly to footer.json file

designerneil committed:
EWL-4657: Updates backstop ref images
Moves hard code in footer.twig into footer.json json
Updates topic.json file to reflect changes done in footer.json

designerneil committed:
Updates header to be set with a level and class so it can be consumed in D8 with needing set these attributes
Restores block overrides

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4657-footer-menu-link-class

# Conflicts:
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_0_bp-xs.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_1_bp-small.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_2_bp-med.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_3_bp-large.png
#	styleguide/backstop_data/bitmaps_reference/backstop_default_topics-page_0_document_4_bp-xl.png
